README.md

```diff
@@ -3,657 +3,288 @@
 Eden is the simplest way to setup & test [EVE](https://github.com/lf-edge/eve)
 and [Adam](https://github.com/lf-edge/adam).
 
-Eden is inspired by Kubernetes workflows and CLI
-
-All components of Eden -- the main executable file `eden`
-and test binaries can be used as stand-alone applications
-without the need to install and configure the development
-environment on your computer. Generally, you will only need
-a text editor to configure the system and create test scenarios
-and scripts using `eden` and pre-compiled test files. To create
-more complex tests, you can use standard Go testing machinery.
+Eden is a management harness that provides two layers of management.
 
-Eden contains series of integration tests implemented in Golang.
-Tests are structured as normal Golang tests by using `_test.go`
-nomenclature and be available for test runs using standard go test framework.
+* infrastructure: deploy and/or delete nodes running [EVE](https://github.com/lf-edge/eve) and a controller [Adam](https://github.com/lf-edge/adam)
+* tasks: execute on EVE, via the controller, one or more tasks
 
-## The architecture of the solution
+Eden is particularly suited to running tests and test suites. These tests must
+meet eden's test API. This repository also includes a framework for simplify
+running the tests on the managed EVE via Adam, and reporting on
+results.
 
-![Architecture](/Eden_eve_architecture.png)
+Eden is inspired by Kubernetes workflows and CLI
 
-You will need another machine besides your edge device to install Eden.
+Note that EVE by itself without a controller is useless in practice. It retrieves its entire
+configuration from the controller, and has no console commands that can be used standalone,
+like general-purpose Linux distributions. You use the controller to tell EVE which workloads
+you want to run. EVE, in turn, runs those workloads in containers or VMs.
 
-This machine should be reachable on the network from the edge device and expose port 3333. This machine will host Adam - the controller, Eden - the control layer with CLI, Redis - the log database, Eserver  - the image/file database for Eve to download from. The alternative is to use some commercial controller.
+Eve supports the following workload formats:
 
-Note that EVE project by itself without a controller is useless in practice. Eve by itself doesn't have any good options/commands to be used as some other OS like Ubuntu. You don't run workloads on Eve kernel. Eve core is separated from apps - the apps are either containers or VMs.
+* OCI image from any OCI compliant registry
+* OS image in qcow2 format
```

Comment:
Seems, it is different formats. OCI image is only about rootfs update (raw is also about rootfs, but also about disk). QCOW2 (also raw, VDI, Parallels) are formats of disk with EVE installed.

Reply:
@giggsoff These are the _workload_ formats, not the _boot_ formats. See L23 right before. It is not authoritative, just to give a feel. EVE definitely supports running OCI containers.

Reply:
got it, thanks for clarification